link_shortener/templates/redirect.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+<!-- Global site tag (gtag.js) - Google Analytics -->
+<script async src="https://www.googletagmanager.com/gtag/js?id={{ ga_id }}"></script>
+<script>
+  window.dataLayer = window.dataLayer || [];
+  function gtag(){dataLayer.push(arguments);}
+  gtag('js', new Date());
+  gtag('config', '{{ ga_id }}');
+</script>
```

Comment:
Zkoušel si tenhle script separovat? Používáš ho na dvou místech, ale jestli to nejde tak nevadí

Reply:
https://github.com/Applifting/link-shortener/pull/25/commits/81f26396dee38be06a3b17a5faa4b2fc1ef0f6c8